client/App.tsx

```diff
@@ -12,28 +33,32 @@ export default function HomePage() {
     const handleSignup = (): void => {
         navigate('/signup');
     };
+    const handleHome = (): void => {
+        navigate('/');
+    };
+    const handleDemo = (): void => {
+        navigate('/demo');
+    };
     const handleAbout = (): void => {
         navigate('/about');
     };
-    const handlePoweredBy = (): void => {
-        navigate('/team');
-    };
 
     return (
         <div>
-            <div id="navbar">
```

Comment:
You are 100% right. The reason why it is there, is because it is just a temp solution, with a FIXME tag above. It will be in a separate file with more functionality. Not even sure the styling is final.